src/cuga/backend/server/main.py

```
@@ -194,6 +199,94 @@ def _skills_effective_enabled() -> bool:
+                    logger.warning(f"Failed to validate and sync policies: {e}")
+            except Exception as e:
+                logger.error(f"Failed to load policies from {cuga_folder}: {e}")
+                app_state.policy_filesystem_sync = None
+        else:
+            logger.info(f"Policy folder {cuga_folder} not found, skipping auto-load")
+            app_state.policy_filesystem_sync = None
+
+        app_state.set_subsystem_status("policy", "ready", "Policy subsystem ready")
+        return True
+
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)